app/views/about/hiring_content.html.haml

```diff
@@ -53,39 +54,48 @@
+        The second role is actually writing educational content.
+        As part of our three year plan, we are extending our content beyond content centred on programming languages, to content about programming in general - looking at fundamental programming concepts, design patterns, and the knowledge that makes someone an excellent programmer.
         You will lead this, planning out and writing the content, supported by both our internal team and our contributor team.
-
-      %p.text-p-xlarge.mb-40
-        Although that's a lot of different hats to wear, they are a synergistic set of responsibilities, which all center around the central mindset of "owning" Exercism's programming content.
-        There will be a lot to learn, but we have a knowledgeable and friendly community who will support you in the role, and three core team members who have all worked in this role and have years of experience to share.
-        We'll give you time to get up to speed and support you in every step of the way.
+      %p.text-p-xlarge.mb-16
+        Thirdly we want to bring our content to life with illustrations, animations and video.
+        While we don't expect you to be an artist or great behind the camera, you'll be largely responsible for finding and liasing with partners to make great content.
+        If you are comfortable recording tutorials alongside the content, that will be a huge bonus too.
 
       %h2.text-h2.mb-12 Is this job right for you?
       %p.text-p-xlarge.mb-12
         The right person for this role will be someone who loves programming, is passionate about education, and who shares our values.
       %p.text-p-xlarge.mb-12
-        In this role you'll be working with a wealth of smart, opiniated volunteers, each with their own preferences and ideas.
-        Discussing and exploring those ideas is a key piece of the role, and something it's important you'll enjoy.
-        At Exercism, we place a high value on
-        %strong pragmatism
-        \- the ability to balance finding the best answer with the one that makes the path forward easiest.
-        To succeed in this role it's important you value that pragmatism too and will enjoy digging into people's opinions, balancing them with your own, and finding pragmatic routes forward.
+        Central to this role is seeing content in the same way our founders do.
+        We want to create content that embodies a sense of playfulness, fun, and approachability.
+        We want our content to feel conversational, maybe even chatty - maybe more like a blog post than a formal teaching article.
+        Articles will ideally include a lot of visuals.
+        Everything we do is centred around making programming approachable, empowering, and never making anyone feel stupid.
```

Comment:
```suggestion
        Everything we do is centred around making programming approachable, empowering, and never making anyone feel lost.
```

Would this make sense?

Reply:
Thanks for the suggestion @lunohodov. There's a longer chat about this above.